src/canvas-extractor/edfi_canvas_extractor/mapping/submissions.py

```diff
@@ -19,12 +19,14 @@ def _get_date_formated(date) -> str:
 
 
 def _get_status(row: pd.Series):
-    if row["late"] == 1:
+    if row["late"] is True:
         return "late"
```

Comment:
This still isn't working. Look at section 103, assignment 114:  `GET 'https://edfialliance.instructure.com/api/v1/sections/103/assignments/114/submissions'` . Submission 11456 was late and is reported as late by the API, but the extractor output says that it is on-time. And Submission 131 says "upcoming" when the status is actually "missing".

I think changing from `== 1` to `is True` was a mistake.

Reply:
even by reverting it, it keeps saying it is on time, I will look into it a little bit more

Reply:
Fixed